File: 2022/12-30/Multiplication Table.py
n,x=list(map(int,input().split()))
sum=0
i=1
while i*i<=x and i*2<=n+1:
    if(x%i==0 and x/i<=n):
        if(i*i==x):
            sum+=1
        else:
            sum+=2
    i+=1
print(sum)

File: 2022/12-30/Multiplication Table.cpp
#include<bits/stdc++.h>
using namespace std;

int main(){
	long n,x;
	long sum=0;
	cin>>n>>x;
	for(long i=1;i<=n;i++){
		if(x%i==0 && x/i<=n){
			sum++;
		}
	}
	cout<<sum<<endl;
	return 0;
}
